### ISSUE TITLE
Slack notifications with color fails

### ISSUE THREAD

**Issue by drwahl** [ansible/awx#2965]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - UI

##### SUMMARY
When using the slack notifier, if a color is defined, the notification will fail with an error from the slack APIs saying "not_in_channel".

##### ENVIRONMENT
* AWX version: 2.7.4
* AWX install method: docker on linux

##### STEPS TO REPRODUCE
Create a slack notification (including the bot user as directed by the AWX/Tower docs).  Then create a slack notification and add #f00 to the "Notification Color" field.

##### EXPECTED RESULTS
A message to be sent to the channel with the defined color.

##### ACTUAL RESULTS
Notification fails with "not_in_channel" error.

**Comment by matburt** [comment]
Is the user actually in the channel? You have to invite the user to the channel you want to receive notifications in.

**Comment by drwahl** [comment]
Yes, the user is actually in the channel. This seems to have to do with inconsistencies in Slack's APIs.

**Comment by andrewsav-bt** [comment]
This seems still to be an issue with the current release of awx.
